cmd/perf-tests.go

```diff
@@ -95,7 +95,6 @@ func selfSpeedTest(ctx context.Context, opts speedTestOpts) (SpeedTestResult, er
 	userMetadata[globalObjectPerfUserMetadata] = "true" // Bypass S3 API freeze
 	popts := minio.PutObjectOptions{
 		UserMetadata:         userMetadata,
-		DisableContentSha256: true,
```

Comment:
This needs to come as a configurable parameter @moting9 - I ended up changing warp https://github.com/minio/warp/pull/292 to have `--disable-sha256-payload`